Fraser/1.99.3/fraser.R

```diff
@@ -0,0 +1,158 @@
+# load FRASER library
+suppressPackageStartupMessages(library(FRASER))
+suppressPackageStartupMessages(library(biomaRt))
+suppressPackageStartupMessages(library(utils))
+
+args <- commandArgs()
+
+
+get_st_rows <- function(input){
+  if(dir.exists(input)){
+    sampleID <- list.files(path=input,pattern="*.bam",full.names=FALSE)
+    bamFile <- list.files(path=input,pattern="*.bam",full.names=TRUE)
+    return(data.table(sampleID,bamFile))
+  }  
+  else if(file.exists(input)){
+    sampleID <- basename(file.path(input))
+    bamFile <- file.path(input)
+    return(data.table(sampleID,bamFile))
+  }
+}
+
+
+create_sample_table <- function(sample,ref,paired_end){
+  sampleTable <- rbind(get_st_rows(sample),get_st_rows(ref))
+  sampleTable <- sampleTable[!grepl(".bai",sampleID), ]
+  sampleTable$group <- c(1:nrow(sampleTable))
+  sampleTable$pairedEnd <- TRUE
+
+  message(date(), ": sampleTable")
+  print(sampleTable)
```

Comment:
Try to avoid print statements, since they might get lost in the stdout. If you want to save / view the data of a variable, write it to a logfile instead